src/app/users/profile/profile.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="container-fluid">
+    <ul class="nav">
+        <li class="nav-item">
+        </li>
+        <li class="nav-item">
+          <button class="btn btn-success" *ngIf="viewerSameAsLoggedInUser()" (click)="editMode()">{{editModeText}}</button>
+          <button class="btn btn-primary" (click)="updateProfile()" *ngIf="inEditMode">Save</button>
+        </li>
+      </ul>
+      <div class="row">
+        <h2 *ngIf="isAssociate()">Associate</h2>
+        <h2 *ngIf="isExpert()">Expert</h2>
+      </div>
+      <div class="row">
+        <div class="col-md-4">
+          <img class="profilePic" src={{user.profilePic}}>
+          <div class="centered">
+            <input type="file" accept="image/*" (change)="onFileChange($event)" *ngIf=inEditMode>
+          </div>
+        </div>
+        <div class="col-md-8">
+          <form [formGroup]="profileForm">
+            <div class="form-group">
+              <label for="profileInputUsername">Username: </label>
+              <span> {{user.username}}</span>
+              <input type="username" formControlName="username" id="profileInputUsername" placeholder="New Username"
+                *ngIf=inEditMode>
+            </div>
+            <div class="form-group">
+              <label for="profileInputEmail">Email: </label>
+              <span> {{user.email}}</span>
+              <input type="email" formControlName="email" id="profileInputEmail" placeholder="New Email" *ngIf=inEditMode>
+            </div>
+            <div class="form-group">
+              <label for="profileInputBio"></label>
+              <span> {{user.bio}}</span>
+              <input type="textarea" formControlName="bio" id="profileInputBio" placeholder="Bio" *ngIf=inEditMode>
```

Comment:
maybe use a textarea element? Or stlye this so it doesn't look like just a text input field.  This can be done in a followup PR if desired